draft-ietf-tsvwg-dtls-over-sctp-bis.md

```diff
@@ -636,15 +644,26 @@ terminated and the associated keying material discarded.
    SCTP-AUTH {{RFC4895}} does not have explicit replay
    protection. However, the combination of SCTP-AUTH's protection of
    DATA or I-DATA chunks and SCTP user message handling will prevent
-   third party attempts to inject or replay SCTP packets resulting in
-   impact on the received protected user message. In fact, this
+   third party attempts to inject or replay SCTP data chunks as long as
+   the Transmission Sequence Numbers (TSNs) are unique. In fact, this
    document's solution is dependent on SCTP-AUTH and SCTP to prevent
    reordering, duplication, and removal of the DTLS records within
    each protected user message.  This includes detection of changes to
    what DTLS records start and end the SCTP user message, and removal of
    DTLS records before an increment to the epoch.  Without SCTP-AUTH,
    these would all have required explicit handling.
 
+   To prevent replay of DATA or I-DATA chunks resulting in impact on the received protected
+   user message, implementations MUST NOT cycle back to the Initial TSN and MUST
```

Comment:
So the SCTP stack will cycle back on the TSN. It is the SCTP-AUTH key that MUST be retired before 2^32 TSN number from the first used with this key.